docs/for-developers/technical-overview.md

```diff
@@ -3,71 +3,200 @@ id: technical-overview
-  
-![Architecture][ac]
-  
-In the following sections, we'll dive deeper into each of the different components of Drips and explain the role that each one plays.
+https://github.com/radicle-dev/radicle-drips-hub
 
-Before diving in however, an important point we would like to make is that Drips is entirely modular and there is no requirement for users to interact with the components in the way they are arranged above. For example, the Drips Webapp is offered only as a reference implementation and users are free to create their own web apps or even interact with the smart contracts directly using transactions issued directly using Etherscan or similar tools. Similarly, there is nothing to stop users from querying the Radicle/Drips subgraph and IPFS files using any applications or tools they desire. By designing Drips in this way, the goal of our team has been to build and offer up the world's most flexible and powerful open protocol for recurring payments and fundraising of any kind. With that in mind, our hope is that many future apps, protocols and tools will be able to build on Drips and use it to empower solutions we can't even imagine today.
+## Drips Concepts
 
-## Accounts and Balances
+### Drips - The Protocol
 
-For a deep dive into the details of how streaming is managed behind the scenes in DripsHub, please see <a href="https://github.com/radicle-dev/radicle-drips-hub/blob/master/docs/how_the_pool_works.md">here</a>.
+Drips is an Ethereum protocol that allows users to set up and manage streaming payments - continuous transfers of funds from one account to
+another over time. We refer to such streaming payments as "Drips". In some ways, streaming payments are similar to traditional recurring payments, but in other ways they are quite different and offer unique advantages.
 
-## Ethereum Contracts
+![](https://i.imgur.com/L3XSop5.png)
 
-### The DaiDripsHub Smart Contract
+Technically, tokens that are Dripped are not sent directly to the recipient's address. Instead the `DripsHub` contract keeps track of the sender and recipient's balances and allows the receiver to collect funds whenever they wish.
 
-<i>DaiDripsHub is part of the <a href="https://github.com/radicle-dev/radicle-drips-hub/blob/master/src/DaiDripsHub.sol">Radicle-Drips-Hub</a> repository.</i>
+### Attributes of Drips
 
-The <a href="https://github.com/radicle-dev/radicle-drips-hub/">DaiDripsHub</a> smart contract is the "lowest-level" component in the Drips architecture. DaiDripsHub provides extremely general, standalone, functionality for allowing any Ethereum address to schedule one-time or streaming payments to
-any other Ethereum address. In this case, "streaming" payments means payments that are made gradually over time -
-something that is not possible using the standard Ethereum transactions.
+A drip is defined by the following attributes
+- **sender** - A Drips user, typically an Ethereum address 
```

Comment:
```suggestion
- **sender** - A Drips user
```

To avoid the confusion with the `AddressApp`

Reply:
This is tough. On one hand we want to be technically precise. On the other hand, we also need to recognize that the hub identities are incredibly technical and will be very unintuitive to must web3 users and developers (where 'users' are almost always Ethereum addresses). This will be even more confusing if in most cases Drips users are actually 1-to-1 with Ethereum addresses in reality anyhow. So with this sort of text, I'm really trying to hint at this complexity while still giving reasonable intuitions to users with different levels technical savvy and different levels of desire to dive into the full details.